src/runtime/hexagon_remote/remote.cpp

```diff
@@ -44,15 +95,239 @@ int halide_do_task(void *user_context, halide_task_t f, int idx,
                    uint8_t *closure) {
     return f(user_context, idx, closure);
 }
+void lock_work_queue() {
+    qurt_mutex_lock(&(work_queue.work_mutex));
+}
+void unlock_work_queue() {
+    qurt_mutex_unlock(&(work_queue.work_mutex));
+}
+void goto_work(work *owned_job) {
```

Comment:
Can you add a comment about what this function does and whether the thread that is calling it owns this job or not? 

It looks like if this is non-null, then this thread owns the job being passed here?